### PR TITLE
Update LayerEncoding.h

### PR DESCRIPTION
**Pull request by tomalin**
Add function to access layer corresponding to given DTC channel. (Needed by L1Trk Future CMSSW code).

### CODE REVIEW

**Review by tschuh — COMMENTED** [review]
Could you please use the at function instead?

File: L1Trigger/TrackerDTC/interface/LayerEncoding.h [hunk 1]
```diff
@@ -23,10 +23,13 @@ namespace trackerDTC {
     ~LayerEncoding() {}
     // decode layer id for given sensor module
     int decode(tt::SensorModule* sm) const;
+    // get encoded layers read by given DTC
+    const std::vector<int>& layers(int dtcId) const {return encodingsLayerId_.at(dtcId%numDTCsPerRegion_);} 
 
   private:
     // helper class to store configurations
     const tt::Setup* setup_;
+    int numDTCsPerRegion_;
```
Comment (tschuh) [inline L1Trigger/TrackerDTC/interface/LayerEncoding.h:33]: Please add comment to new data member.

Comment (tomalin) [reply L1Trigger/TrackerDTC/interface/LayerEncoding.h:33]: done

**Comment by tomalin** [comment]
> Could you please use the at function instead?

done

File: L1Trigger/TrackerDTC/interface/LayerEncoding.h (hunk 1 above)
Comment (tschuh) [reply L1Trigger/TrackerDTC/interface/LayerEncoding.h:33]: when you mean processing nonant I expect the value to be 48 and when you mean detector nonant I expect 24, which one is it?

Comment (tomalin) [reply L1Trigger/TrackerDTC/interface/LayerEncoding.h:33]: A detector phi nonant = 24. Now clarified comment.
It corresponds to Setup::numDTCsPerRegion()

**Review by tschuh — APPROVED** [review]

**Pull request merged**